### ISSUE TITLE
is training via ListTrainer still possible?

### ISSUE THREAD

**Issue by lordskyzw** [gunthercox/chatterbot-corpus#241]
when i try to employ ListTrainer i get a "no object named 'find' " error

**Comment by gunthercox** [comment]
Yes, this is still possible in the latest version and there are no plans to drop support for the `ListTrainer` at the current time.

Usage documentation: https://docs.chatterbot.us/training/#training-via-list-data
